core/tests/protractor/accessibility.js

```diff
@@ -41,6 +61,364 @@ describe('screenreader and keyboard user accessibility features', function() {
         .getAttribute('id'));
   });
 
+  it('should test the navigation shortcut ctrl+0',
```

Comment:
Typically the `it` string parameter should make sense if it was part of a phrase with the parent `describe` block. Here, the phrase "screenreader and keyboard user accessibility features should test the navigation shortcut ctrl+0", doesn't really make sense.

Maybe the `describe` string should be something along the lines of 'keyboard shortcuts' and each `it` string should be something like 'should navigate to X page when Y is pressed'

Reply:
done